### PR TITLE
fix: add subscription id to json editor schema

### PR DESCRIPTION
**Pull request by zidaneymar**
## Description
add subscription id to json editor schema

## Task Item
#minor

### CODE REVIEW

File: extensions/azurePublish/src/node/schema.ts [hunk 1]
```diff
@@ -30,6 +30,10 @@ const schema: JSONSchema7 = {
       title:
         'Runtime identifier for hosting bot, default to win-x64, please refer to https://docs.microsoft.com/en-us/dotnet/core/rid-catalog',
     },
+    subscriptionId: {
+      type: 'string',
+      title: 'id of your subscription'
+    },
     settings: {
```
Comment (luhan2017) [inline extensions/azurePublish/src/node/schema.ts:37]: Upper case I

Comment (luhan2017) [reply extensions/azurePublish/src/node/schema.ts:37]: I found that QnA is also missing from the schema, let's add it.

Comment (zidaneymar) [reply extensions/azurePublish/src/node/schema.ts:58]: > I found that QnA is also missing from the schema, let's add it.

added

File: extensions/azurePublish/src/node/schema.ts [hunk 2]
```diff
@@ -30,6 +30,31 @@ const schema: JSONSchema7 = {
...
+    },
+    resourceGroup: {
+      type: 'string',
+      title: 'Name of your resource group'
+    },
+    botName: {
+      type: 'string',
+      title: 'Name of your bot channel registration service'
+    },
+    abs: {
```
Comment (luhan2017) [inline extensions/azurePublish/src/node/schema.ts:45]: QnA is missing from schema and there is no abs section now, all of the settings from abs are under root.

**Comment by benbrown** [comment]
It would be great if this PR could include all necessary schema updates for the settings!

File: extensions/azurePublish/src/node/schema.ts (hunk 2 above)
Comment (zidaneymar) [reply extensions/azurePublish/src/node/schema.ts]: qna and other resources added.

**Review by luhan2017 — APPROVED** [review]

**Pull request merged**